front/src/modules/companies/constants/companiesAvailableFieldDefinitions.tsx

```diff
@@ -179,4 +180,17 @@ export const companiesAvailableFieldDefinitions: ColumnDefinition<FieldMetadata>
       isVisible: false,
       infoTooltipContent: 'The company Twitter account.',
     } satisfies ColumnDefinition<FieldURLMetadata>,
+    {
+      fieldMetadataId: 'customizeFields',
```

Comment:
@umrkhn Let's avoid hacking into the availableFieldDefinitions constants :) What we want here is a hard coded option within RecordTableHeaderPlus button.

We actually have refactored this section and this file doesn't exist anymore anyway as we are computing the available fields dynamically now!